app/models/ban.rb

```diff
@@ -58,7 +58,7 @@ def self.search(params)
     when "expires_at_desc"
       q = q.order("bans.expires_at desc")
     else
-      q = q.order("bans.id desc")
+      q = q.default_order
```

Comment:
I removed the `return q if params.blank?` from this method in 5ea61c69212d56bbf301ba87cd689d312ea09802, it would have to be added back.